chainerx_cc/chainerx/cuda/cuda_device/hyperbolic.cu

```diff
@@ -0,0 +1,34 @@
+#include "chainerx/cuda/cuda_device.h"
+
+#include <cmath>
+#include <cstdint>
+
+#include <cuda_runtime.h>
+
+#include "chainerx/array.h"
+#include "chainerx/cuda/cuda_runtime.h"
+#include "chainerx/cuda/cuda_set_device_scope.h"
+#include "chainerx/cuda/elementwise.cuh"
+#include "chainerx/cuda/kernel_regist.h"
+#include "chainerx/cuda/numeric.cuh"
+#include "chainerx/device.h"
+#include "chainerx/dtype.h"
+#include "chainerx/kernels/hyperbolic.h"
+#include "chainerx/numeric.h"
+#include "chainerx/scalar.h"
```

Comment:
Let's do these inclusions in the `kernel_regist.h`s. Maybe in a separate PR.

Reply:
I'll fix them in another PR.